src/models/PrimaryAuth.js

```diff
@@ -37,9 +37,24 @@ function (Okta, BaseLoginModel, CookieUtil, Enums) {
       }
 
       return {
-        username: ['string', true, username],
+        username: {
+          type: 'string',
+          validate: function (value) {
+            if(!value || value.length < 1) {
+              return 'Please enter a username';
```

Comment:
I think `validate` is good, please change to use i18n, and also add a test to verify `new Model({username: ''}).validate()` will return `{'username' :  '{{your message}}'}`.